anchor/common/qbft/src/lib.rs

```
@@ -621,7 +633,14 @@ where
                     return false;
                 }
 
-                if !self.check_quorum(&round_change.round_change_justification) {
+                // Deserialize prepare justifications for validation
+                let signed_inner_rc_justifications: Vec<SignedSSVMessage> = round_change
+                    .round_change_justification
+                    .iter()
+                    .filter_map(|bytes| SignedSSVMessage::from_ssz_bytes(bytes).ok())
```

Are errors during deserialization being silently ignored?

---

What's the resolution for this?

---

Now it's using a map, I guess. It's weird, I can't see these conversations in the code.